### PR TITLE
Add missing link

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Adds a missing link to bot.route.

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
@roj1512 friendly reminder

**Comment by roj1512** [comment]
Thanks!

**Review by roj1512 — APPROVED** [review]

File: site/docs/advanced/structuring.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ That being said, a straightforward and proven strategy to structure your code is
    Every single one of these parts exposes middleware that will handle the designated messages.
 2. Create a bot instance centrally that merges all middleware by installing it onto the bot.
 3. (Optional.) Pre-filter the updates centrally, and send down updates the right way only.
-   You may also want to check out `bot.route` ([API Reference](https://doc.deno.land/https://deno.land/x/grammy/mod.ts/~/Composer#route)) or alternatively the [router plugin](../plugins/router.md) for that.
+   You may also want to check out `bot.route` ([API Reference](https://deno.land/x/grammy/mod.ts?s=Composer#method_route_0)) or alternatively the [router plugin](../plugins/router.md) for that.
```
Comment (CikiMomogi) [inline site/docs/advanced/structuring.md:23]: This link appears to be broken, so I made some adjustments to it. Could you please verify that the new one is correct?

Comment (KnorpelSenf) [reply site/docs/advanced/structuring.md:23]: It is correct.

**Pull request merged**